pkgs/by-name/di/dipc/package.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "depc";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "doprz";
+    repo = "dipc";
+    rev = "bf578bd9474084b7099ef665138667e486dce671";
+    hash = "sha256-RXEC8bwdnUOaDmYIb7ci/JD+vi16tBn55FRsUmwaRzk=";
+  };
+
+  cargoHash = "sha256-t+G8eQufZPge9vyz6jdET7PZ76eQkCyPt0m2CYzIuD0=";
+
+
+  meta = with lib; {
+    description = "Convert your favorite images and wallpapers with your favorite color palettes/themes ";
+    homepage = "https://github.com/doprz/dipc";
+    license = licenses.free;
```

```suggestion
    license = with licenses; [ mit free ];
```

not actually entirely sure how to model this correctly